apps/ledger-live-mobile/src/families/celo/accountActions.tsx

```diff
@@ -44,6 +44,12 @@ const getMainActions = ({
       navigationParams,
       label: <Trans i18nKey="account.stake" />,
       Icon: () => <CeloIcon isDisabled={false} />,
+      event: "button_clicked",
+      eventProperties: {
+        button: "stake",
+        token: "CELO",
+        page: "Account Page",
```

Comment:
Will we call these actions from other pages? If so, this `page` would be wrong

Reply:
Ok

Reply:
Good eye! not because this track will be triggered when we click the button. We will use from here only the navigation for the common stake flow.